deploy/crds/crd-certificates.yaml

```diff
@@ -67,6 +67,21 @@ spec:
                 - issuerRef
                 - secretName
               properties:
+                additionalOutputFormats:
+                  description: AdditionalOutputFormats allows for requests for additional formats in which the private key should be written to the secret
```

Comment:
We should make a note in the comments of the API (so it is displayed here) that this feature is in Alpha, and if the user wants to use it, they will need to enable the feature gate else it won't do anything.

Reply:
Agree, comment added.